webpki/Cargo.toml

```diff
@@ -64,10 +64,10 @@ trust_anchor_util = ["std"]
 std = []
 
 [target.'cfg(target_arch = "wasm32")'.dependencies]
-ring_wasmable = { package = "ring", path = "../ring", version = "0.16.0", default-features = false, features = ["alloc"] }
+ring_wasmable = { package = "ring", git = "https://github.com/Niederb/ring-xous.git", branch = "0.16.20-cleanup", default-features = false, features = ["alloc"] }
```

Comment:
Not sure about how the forks are/should be handled, but I think it would be better to have the repository in the `scs` namespace, rather than in individual dev's.

Reply:
If i remember correctly I tried that but the thing is that each user/organization can have only one fork and there is already https://github.com/scs/ring-sgx and there is also https://github.com/scs/ring-nostd/. So I couldn't do that. 
Anyway I think it would make more sense to have it in integritee. There is also already a `ring` fork there, but this one is from me and could be deleted.